README.md

```diff
@@ -262,11 +263,12 @@ Configuration is done by user application: it should configure exporter and may
     {
         builder
             .SetSampler(Samplers.AlwaysSample)
-            .UseZipkin(o => o.ServiceName = "my-service")
+            .UseZipkin()
 
             // you may also configure request and dependencies collectors
             .AddRequestCollector()
-            .AddDependencyCollector())
+            .AddDependencyCollector()
+            .MergeResource(new Resource(new Dictionary<string, string>() { { "service.name", "my-service" } }))

```

Comment:
```suggestion
            .SetResource(new Resource(new Dictionary<string, string>() { { "service.name", "my-service" } }))
```

Reply:
Nope. User intent here is to add couple properties, not to override the default one.